shenyu-admin/src/main/resources/sql-script/oracle/schema.sql

```diff
@@ -0,0 +1,2371 @@
+/* 删除表空间 */
```

Comment:
add apache license and trsanfrom cn to en

Reply:
done